src/session/content/chat_action_bar.rs

```diff
@@ -270,6 +290,27 @@ impl ChatActionBar {
         }
     }
 
+    pub(crate) fn handle_paste_action(&self) {
+        if let Some(chat) = self.chat() {
+            let display = gdk::Display::default().unwrap();
+            let clipboard = display.clipboard();
+            clipboard.read_texture_async(
```

Comment:
Mh, thinking about that we don't really have to do any image operation like we're doing here. We just need the actual clipboard data, without trying to decode the image and then reencoding it to a png.